hack/mesh.sh

```diff
@@ -11,12 +11,10 @@ source "$(dirname "${BASH_SOURCE[0]}")/lib/__sources__.bash"
 
 set -Eeuo pipefail
 
-exitcode=0
-
 if [[ $UNINSTALL_MESH == "true" ]]; then
-  (( !exitcode )) && uninstall_mesh || exitcode=1
+  (( !exitcode )) && uninstall_mesh || exit 1
```

Comment:
I'm curious if this will work. The exitcode variable is still on this line but its definition has been removed.

Reply:
Thank you! Updated. It was my bad.